core-ui/src/resources/Functions/FunctionCreate.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
 import * as jp from 'jsonpath';
+import * as _ from 'lodash';
```

Comment:
I believe there's a bit of a push towards using
```suggestion
import { cloneDeep } from 'lodash';
```
instead - makes for a somewhat cleaner code.
It's not done so everywhere, but I think it's good practice to at least use it like this in new places.